tests/components/modbus/test_sensor.py

```diff
@@ -535,7 +535,7 @@ async def test_config_wrong_struct_sensor(
             },
             [0x3730, 0x302D, 0x2D35, 0x3032, 0x3032, 0x3120, 0x3A34, 0x3533],
             False,
-            "07-05-2020 14:35",
+            "07-05-2020 14:35",  # FAIL
```

Comment:
What do all these `# FAIL` comments have to do with config flow?

Reply:
My BAD, apologies !! I think it is a left over from another test.

All removed.